src/sdl2/controller.rs

```diff
@@ -338,6 +352,7 @@ pub struct GameController {
 
 impl GameController {
     #[inline]
+    #[doc(alias = "SDL_GameControllerName")]
```

Comment:
This one matched the wrong method, the right one is a few lines below.

Reply:
Oh fun bug: didn't take into account one line functions haha.